source/Patches/AttackStackSequence_OnAttackComplete.cs

```diff
@@ -0,0 +1,134 @@
+﻿using BattleTech;
+using BattleTech.BinkMedia;
+using Org.BouncyCastle.Math.Raw;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CustomSalvage
+{
+    [HarmonyPatch(typeof(AttackStackSequence), "OnAttackComplete")]
```

Comment:
use nameof(AttackStackSequence.OnAttackComplete) to allow ctrl-click (Rider) to open the dissassembler directly.. makes faster to check what is being patched. also allows static analysis on if there is a typo